imap_processing/hit/l2/hit_l2.py

```diff
@@ -1,26 +1,30 @@
 """IMAP-HIT L2 data processing."""
 
 import logging
+from pathlib import Path
 from typing import NamedTuple
 
 import numpy as np
 import pandas as pd
 import xarray as xr
 
-from imap_processing import imap_module_directory
 from imap_processing.hit.hit_utils import (
     add_energy_variables,
     get_attribute_manager,
     initialize_particle_data_arrays,
     sum_particle_data,
 )
 from imap_processing.hit.l2.constants import (
+    L2_STANDARD_ANCILLARY_PATH_PREFIX,
+    L2_SUMMED_ANCILLARY_PATH_PREFIX,
     STANDARD_PARTICLE_ENERGY_RANGE_MAPPING,
 )
 
 logger = logging.getLogger(__name__)
 
-# TODO review logging levels to use (debug vs. info)
+# TODO:
```

Comment:
Ignore this if you want: If you put the TODO on both lines then it'll be easier to view in an IDE TODO viewing tool